src/Containers/TriggerAddContainer.jsx

```diff
@@ -150,6 +150,7 @@ class TriggerAddContainer extends React.Component<Props, State> {
                 }
                 try {
                     const { id } = await moiraApi.addTrigger(trigger);
+                    const history = useHistory();
```

Comment:
@ani4aniket it's okay that hook uses inside `if` statement?

Reply:
Because https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Reply:
@sashasushko, You are right, Should I use the normal `history` prop inside `if` statements?
Also, we need to add `rules-of-hooks` eslint config to avoid such use cases.
I will create another PR to fix this. Is this ok?

Reply:
I think, you can use hook, but declare variable in the top of method

Reply:
>  Also, we need to add rules-of-hooks eslint config to avoid such use cases.

Yeah, and other rules too. Maybe, I'll add issue about it.

Reply:
@sashasushko I have committed the changes, you have suggested also added `rules-of-hooks` to eslint config. Please check it out. Thanks